chromium_src/chrome/browser/resources/side_panel/customize_chrome/app.ts

```diff
@@ -36,3 +40,81 @@ declare global {
 }
 
 customElements.define(ClosePanelButton.is, ClosePanelButton)
+
+
+// @ts-expect-error We're adding a custom property to the AppElements for hiding the "Darker theme" toggle.
+AppElement.prototype.shouldShowDarkerThemeToggle_ = loadTimeData.getBoolean('shouldShowDarkerThemeToggle');
```

Comment:
I think you should be able to do this without the `@ts-expect-error`

https://github.com/brave/brave-core/blob/master/docs/webui_overriding.md#lit-functions

Reply:
Yup, absolutely. Done.